Tensile/KernelWriterAssembly.py

```diff
@@ -6195,6 +6246,12 @@ def globalWriteBatch(self, kernel, beta, edge, lsu, atomic, gwvw, atomicW, \
             else: # HPA
               kStr += inst("v_mul_f32", vgpr("ValuC+%u"%sumIdxV), sgpr("Alpha"), vgpr("ValuC+%u"%sumIdxV), "*= alpha")
 
+          elif kernel["ProblemType"]["DataType"].isInt8x4():
+            # below assume we use s_mul_i32 not v_mul_i32_i24. this assumes accuracy more important then speed. Is this a correct assumption?
+            kStr += inst("s_mul_i32", vgpr("ValuC+%u"%sumIdxV), sgpr("Alpha"), vgpr("ValuC+%u"%sumIdxV), "*= alpha" )
+            if self.db["CheckValueC"] >= 0:
+              kStr += self.assert_eq(vgpr("ValuC+%u"%sumIdxV), 0.0)
+
```

Comment:
is it correct to use s_mul_i32 here in place of v_mul_f32. I assume we do not want v_mul_i32_i24